HarmonyCore.Test/Bridge/BasicBridge.dbl

```diff
@@ -33,7 +33,7 @@ namespace HarmonyCore.Test.Bridge
             
             if(Environment.OSVersion.Platform == PlatformID.Unix) then
             begin
-                data contextPool = new ExternalContextPool<T>(Path.Combine(actualDblDir, "bin/dbs"), actualDBRPath, testDirFolder, ^null, 4) 
```

Comment:
`actualDbrDir` is intended to allow overriding the base path as a parameter, can you instead fit this into the default value mechanism we have above?

Reply:
I updated it to use actualDblDir.